src/Routes.js

```diff
@@ -2,18 +2,18 @@
 const routes = {
   HomePage: {
     url: '/',
-    component: 'src/pagesComponents/HomePage.js',
+    component: 'src/pages/HomePage.js',
     seo: { title: 'Home', description: '' },
   },
   AboutPage: {
     url: '/about/',
-    component: 'src/pagesComponents/AboutPage.js',
+    component: 'src/pages/AboutPage.js',
     seo: { title: 'About', description: '' },
   },
-  WhatWeDoPage: {
-    url: '/what-we-do/',
-    component: 'src/pagesComponents/WhatWeDoPage.js',
-    seo: { title: 'What we do', description: '' },
+  ModelPage: {
+    url: '/model/',
```

Comment:
In this routing setup, do these URLs work with both the trailing slash and non-trailing slash versions?

(do `/model` and `/model/` both resolve to the page correctly?)

Reply:
Both resolve correctly, so I will remove the last slash :smile:

Reply:
I don't care either way, I just wanted to make sure they resolve correctly.